src/main/java/ch/njol/skript/Skript.java

```diff
@@ -680,13 +681,20 @@ protected void afterErrors() {
 				debug("Early init done");
 
 				if (TestMode.ENABLED) {
-					if (TestMode.DEV_MODE) {
+					if (!TestMode.DEV_MODE) {
 						runTests(); // Dev mode doesn't need a delay
 					} else {
-						Bukkit.getWorlds().get(0).getChunkAtAsync(100, 100).thenRun(() -> runTests());
+						World world = Bukkit.getWorlds().get(0);
+						if (isPaper()) {
+							world.getChunkAtAsync(100, 100).thenRun(() -> runTests());
+						} else {
+							world.getChunkAt(100, 100);
```

Comment:
Skript shades in PaperLib, and PaperLib has a getChunkAsync method which would run asynchronous on paper, and sync on Spigot